src/librustc_codegen_llvm/builder.rs

```diff
@@ -411,6 +411,48 @@ impl BuilderMethods<'a, 'tcx> for Builder<'a, 'll, 'tcx> {
         }
     }
 
+    fn nswadd(&mut self, lhs: &'ll Value, rhs: &'ll Value) -> &'ll Value {
+        self.count_insn("nswadd");
```

Comment:
Consider copying the codegen test from https://github.com/rust-lang/rust/pull/52205/files#diff-ecdf74312d2f8c6c057ef9070b38bcaa

Reply:
ok